tests/e2e/cucumber/column-picker.feature

```diff
@@ -1,14 +1,14 @@
 Feature: Column picker component
 
   Scenario Outline: 1. Column picker component allows to select number of rows in the grid
-    Given start page
-    Then  start button is clicked
-    When  a "<query>" is typed
+    Given start page with "<view>" size view
+    When  start button is clicked
+    And   a "<query>" is typed
     Then  grid shows 4 results per row
     When  value <columns> from column picker list is clicked
     Then  column picker list value <columns> is selected
     And   grid shows <columns> results per row
     Examples:
-      | query   | columns |
-      | shirt   | 6       |
-      | shirt   | 0       |
+      | query   | columns | view        |
+      | shirt   | 6       | macbook-13  |
+      | shirt   | 0       | macbook-13  |
```

Comment:
```suggestion
      | shirt   | 0       | iphone-7  |
```

Reply:
There is no column-picker for mobile, so actually in this case there should be only one row:
```
    Examples:
      | query   | columns | view               |
      | shirt     | 6             | macbook-13  |
```